conversion_test.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2020 Martin Holst Swende. All rights reserved.
```

Comment:
I guess we could remove my explicit name, and use AUTHORS instead. I kept it in my name originally, so I would be sure that there would be no problems to change the license at a later point. I guess we're good with the current license?

Reply:
We can handle that separately some other time. I this case I copied code from another file so I used the same one.

I'm fine with the license. My default one is Apache2, but I rather don't care witch open-source license it is.